Kajabi/sage-lib
# Sage Hero - add Rails component

## Description

- [x] - create Sage Rails hero component
- [x] - add js functionality that allows the hero to self dismiss

### Screenshots


|  mobile  |  desktop  |
|--------|--------|
|![Screen Shot 2020-12-10 at 8 22 01 PM](https://user-images.githubusercontent.com/1241836/101853578-6f2d4a80-3b25-11eb-9fb7-0c24bea71c01.png)|![Screen Shot 2020-12-09 at 5 05 30 PM](https://user-images.githubusercontent.com/1241836/101699488-d1604f80-3a40-11eb-924b-ccf09117b4f1.png)|


### Specs
[Beta Features page -> Hero](https://www.figma.com/file/o8hxGSv1fEVPVm4oSF3c92/Beta-Features?node-id=149%3A381)
## Test notes



### Steps for testing
1. Visit the Sage Rails Hero page: http://localhost:4000/pages/object/hero
2. Verify that it matches the spec
3. Click the `dismiss` button and be sure the it removes itself from the DOM
